README.md

```diff
@@ -81,11 +81,12 @@ The following dependencies are required to build it:
 The following dependencies enable various optional features:
 - bash-completion
 
-It can be built and installed as any other typical Meson-based project:
+It can be built, installed and uninstalled as any other typical Meson-based project:
 ```console
 [user@hostname toolbox]$ meson -Dprofile_dir=/etc/profile.d builddir
 [user@hostname toolbox]$ ninja -C builddir
 [user@hostname toolbox]$ sudo ninja -C builddir install
+[user@hostname toolbox]$ sudo ninja -C builddir uninstall
```

Comment:
Is this really necessary?

These steps are only meant to be a quickstart guide, in the same style as Meson's own [quickstart guide](https://mesonbuild.com/Quick-guide.html), and not meant to document all the possibilities. Note how it doesn't mention the `--prefix` command line option, which is something almost everybody would be using.

Other than that it seems that Meson's `uninstall` target is still a [bit rough](https://github.com/mesonbuild/meson/issues/1974). eg., while `ninja uninstall` works, `meson uninstall` doesn't seem to work, even though it looks like it's supposed to.